.travis.yml

```diff
@@ -3,9 +3,7 @@ language: php
 sudo: false
 
 php:
-  - 7.1
```

Comment:
I think we should still test against older versions no?

Reply:
Yes, this PR is a test. I was not going to wait for other version to run.